packages/sit-onyx/src/components/OnyxBottomBar/OnyxBottomBar.ct.tsx

```diff
@@ -73,12 +73,12 @@ Object.entries(ONYX_BREAKPOINTS).forEach(([breakpoint, width]) => {
 
     /* eslint-disable playwright/no-conditional-in-test, playwright/no-conditional-expect -- prevent useless screenshots for breakpoints that don't have a max width */
     if (width > ONYX_BREAKPOINTS.md) {
-      const app = page.locator(".onyx-app");
+      const gridLayout = page.locator(".onyx-grid-layout");
 
-      await app.evaluate((element) => element.classList.add("onyx-grid-max-md"));
+      await gridLayout.evaluate((element) => element.classList.add("onyx-grid-max-md"));
```

Comment:
Are the onyx-grid-max-md and onyx-grid-center classes only working when applied to the onyx-grid-layout? Usually they are defined once on app level so it should work to apply them onyx the root element :)